tools/WrapExecutableLinker.sh

```diff
@@ -44,3 +44,19 @@ else
     "$@" -DGIT_COMMIT_HASH=$git_commit_hash -DGIT_BRANCH=$git_branch \
          "${InfoAtLink_file}"
 fi
+
+if @WRAP_EXECUTABLE_LINKER_USE_STUB_OBJECT_FILES@; then
+    # When linking, mv the object files to a temporary file (this
+    # preserves times), then use touch to create a new file empty
+    # stub object file with the same times (atime and mtime) as
+    # the original object file, except that it is 0kB
+    for file in "$@"
+    do
+        if [[ $file =~ .*\.cpp\.o ]]; then
+            temp_file="${file}.tmp_to_eliminate_size"
+            mv ${file} ${temp_file}
+            touch -r ${temp_file} ${file}
+            rm ${temp_file}
```

Comment:
This script is at least mostly quoting things correctly, so quote the variables in this section, too.